manifests/templates/logsearch/logsearch-infrastructure-warden.yml

```diff
@@ -0,0 +1,463 @@
+---
+meta:
+  environment: (( merge ))
+
+  stemcell:
+    name: bosh-warden-boshlite-ubuntu-trusty-go_agent
+    version: latest
+
+properties:
+  <<: (( merge ))
+  elasticsearch:
+    <<: (( merge ))
+    exec:
+      <<: (( merge ))
+      environment:
+        <<: (( merge ))
+        ES_HEAP_SIZE: 256M
+
+update:
+  canaries: 1
+  max_in_flight: 50
+  canary_watch_time: 1000-600000
+  update_watch_time: 1000-600000
+  serial: false
+
+compilation:
+  cloud_properties:
+    name: random
+
+resource_pools:
+- name: tiny
+  cloud_properties:
+    name: random
+- name: medium
+  cloud_properties:
+    name: random
+- name: high_memory
+  cloud_properties:
+    name: random
+- name: high_cpu
+  cloud_properties:
+    name: random
+
+networks: (( merge || networks_default ))
+networks_default:
```

Comment:
This is indeed really confusing. This is actually a workaround of a issue in the warden bosh CPI, which forces you to define all those networks.

This is only required for warden, and we don't really use it, unless you deploy this on bosh-lite.